CHANGELOG.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 master
 ------
 
-- (`#103 <https://github.com/GranthamImperial/silicone/pull/101>`_) Update github address to GranthamImperial.
+- (`#108 <https://github.com/GranthamImperial/silicone/pull/108>`_) Added a multiple infiller to split up an aggregate with a remainder. Disabled test for downloading database.
```

Comment:
Did you disable the test or just make it not fail if you get a runtime error?

Reply:
The latter, the notebooks will still fail if the database is down anyway.

Reply:
good change then. Maybe we should rethink notebook testing to avoid those failures (but I guess we only do it if helpful, no need to add extra jobs without reason)